tbmalt/io/skf.py

```diff
@@ -749,6 +751,215 @@ def from_dir(cls, source: str, target: str):
             This map file example shows that there are two compression radii
             used for "X" and two for "Y", giving a total of four combinations.
             It is therefore expected that there are four sk-files.
+            If the source is 'skf' type, the code will loop over files like:
```

Comment:
There are two very different parsing methods present. I think we need to separate the two methods out into two different functions and call one `from_dir_with_map` and the other one (your one) `from_dir`. Otherwise this might end up confusing users having two different behaviors.